apps/portal/src/content/docs/components/tooltip.mdx

```diff
@@ -11,9 +11,10 @@ import { DocsPage } from "../../../components/docs-page";
   client:only
   code={`<Tooltip.Provider>
     <Tooltip.Root>
-        <Tooltip.Trigger>Hover me</Tooltip.Trigger>
+        <Tooltip.Trigger><Button theme="default">New Connector</Button></Tooltip.Trigger>
```

Comment:
This will put a button in a button. When putting a `Button` or `Link` into a `Trigger`, make sure you set `asChild` on the `Trigger`
```suggestion
        <Tooltip.Trigger asChild>
            <Button theme="default">New Connector</Button>
        </Tooltip.Trigger>
```